pkgs/development/python-modules/requests/default.nix

```diff
@@ -4,11 +4,11 @@
 
 buildPythonPackage rec {
   pname = "requests";
-  version = "2.23.0";
+  version = "2.24.0";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1rhpg0jb08v0gd7f19jjiwlcdnxpmqi1fhvw7r4s9avddi4kvx5k";
+    sha256 = "06r3017hz0hzxv42gpg73l8xvdjbzw7q904ljvp36b5p3l9rlmdk";
```

Comment:
this is on staging, and the cause of your mass rebuilds. I would drop the commit if you can.

If you can't then I would wait until the next staging cycle gets merged

Reply:
I see. Would it be possible to patch the pypi packages somehow to make them depend on requests==2.23.0 instead?

Reply:
Ok, commit dropped.